.github/workflows/build-gh-pages.yml

```diff
@@ -43,3 +43,32 @@ jobs:
           git add -A
           git commit -m "update with results of commit ${{ github.sha }}"
           git push -f origin gh-pages
+      - name: Deploy ssh key for scratch site (pull request)
+        if: ${{ github.event_name == 'pull_request' && github.ref_name != 'master' && github.repository == 'OSGeo/spatialreference.org' }}
+        shell: bash -l {0}
+        run: |
+          sudo apt install hxtools
+          mkdir $HOME/.ssh
+          # Note: this is obviously unsecure !!! But we don't mind if the content of sr-org-scratch/sr-org-scratch is "compromised"
+          cat .github/workflows/some_secret.txt | rot13 > $HOME/.ssh/id_rsa
```

Comment:
Is there any reason to not use a proper secret, as a few lines above?

Reply:
> Is there any reason to not use a proper secret, as a few lines above?

github secrets are only working on push, not pull requests